update way of explaining how to add methods to prototypes (#18336)
mdn/content: files/en-us/learn/javascript/objects/object_prototypes/index.md

  this.name = name;
}

Object.assign(Person.prototype, personPrototype);
// or
// Person.prototype.greet = personPrototype.greet;
```

Here we create:

- an object `personPrototype`, which has a `greet()` method
- a `Person()` constructor function which initializes the name of the person to create.

> We then set the `Person` function's `prototype` property to point to `personPrototype` using [Object.assign](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) to put a new function inside the prototype.

Reviewer: ```suggestion
We then put the methods defined on `personPrototype` onto the `Person` function's `prototype` property using [Object.assign](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign).
```

> After this code, objects created using `Person()` will get `personPrototype` as their prototype.

Reviewer: ```suggestion
After this code, objects created using `Person()` will get `Person.prototype` as their prototype, which automatically contains the `greet` method.
```
